internal/tableprinter/tableprinter_options.go

```diff
@@ -0,0 +1,17 @@
+package tableprinter
+
+type TableOutputFormat string
+
+const (
+	OutputFormatTable = TableOutputFormat("table")
+	OutputFormatJSON  = TableOutputFormat("json")
+	OutputFormatYAML  = TableOutputFormat("yaml")
+)
+
+type TablePrinterOption func(*TablePrinterOptions)
+
```

Comment:
Let's add an additional option `WithIsTTY(istty bool)` that sets the internal value for `istty`. Also please can you lowecase this internal attribute. This will allow us to prevent use of `iostreams` within the package.

Reply:
Done 👍🏽